doc/classes/ScriptEditor.xml

```diff
@@ -57,6 +57,13 @@
 				[b]Note:[/b] Does not apply to scripts that are already opened.
 			</description>
 		</method>
+		<method name="reload_open_files">
+			<return type="void" />
+			<param index="0" name="refresh_only" type="bool" default="false" />
+			<description>
+				Reload all currently opened files from disk in case the file contents are newer.
```

Comment:
```suggestion
				Reload all currently opened files if [param refresh_only] is [code]false[/code]. When [param refresh_only] is [code]true[/code], only modified files since its last save are reloaded. 
				[b]Note:[/b] This method has no effect if an external editor is currently active.
```
This just clarifies the description of the method's internals, and how the argument is interpreted.

Reply:
great addition thank you 👍